### PR TITLE
Add Zoom meeting passcode

### PR DESCRIPTION
**Pull request by mtwo**
This is a documentation-only change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ availability. Check the [OpenTelemetry community
 calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com)
 for specific dates.
 
-Meetings take place via [Zoom video conference](https://zoom.us/j/8287234601).
+Meetings take place via [Zoom video conference](https://zoom.us/j/8287234601). The passcode is _77777_.
```
Comment (reyang) [inline README.md:86]:
```suggestion
Meetings take place via [Zoom video conference](https://zoom.us/j/8287234601). The passcode is `77777`.
```

**Review by reyang — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ availability. Check the [OpenTelemetry community
 calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com)
 for specific dates.
 
-Meetings take place via [Zoom video conference](https://zoom.us/j/8287234601).
+Meetings take place via [Zoom video conference](https://zoom.us/j/8287234601). The passcode is `77777`.
```
Comment (Oberon00) [inline README.md:86]: Do you intentionally not use the link with embedded passcode?

**Comment by mtwo** [comment]
Correct, I don’t want it to get scraped by a bot

________________________________

File: README.md (hunk 2 above)
Comment (cijothomas) [inline README.md:86]:
```suggestion
Meetings take place via [Zoom video conference](https://zoom.us/j/8287234601).
The passcode is `77777`.
```

**Pull request merged**